lib/timer_count_down.dart

```diff
@@ -1,9 +1,8 @@
 import 'dart:async';
 
+import 'package:elhariry/Widgets/Timer/timer_controller.dart';
```

Comment:
Please, make sure to use `import 'package:timer_count_down/timer_controller.dart';`

Reply:
because I used the package outside its main Flutter packaging folder.

Reply:
okay, before merging, please, remove this

Reply:
I have, you can check it out